src/validate.js

```diff
@@ -1,13 +1,15 @@
 import markdownlint from "markdownlint";
+import yaml from 'js-yaml';
 import markdownlintGitHub from "@github/markdownlint-github";
 
-export const validate = (markdown) => 
-  (markdownlint
+export const validate = (markdown, config) => {
+  const configObject = yaml.load(config)
```

Comment:
does this function handle if the extra custom config doesn't exist?

Reply:
Sure does! I have a test repo that I will invite you too.